<launch>
    <!-- Starte den usb_cam Node -->
    <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen">
        <!-- Übergebe die Parameter für das Video-Gerät und den Kameranamen -->
        <param name="video_device" value="/dev/video0"/>
        <param name="camera_name" value="usb_cam"/>
    </node>
    
    <!-- Starte den hsv_camera_feed Node -->
    <node pkg="computer_vision" type="hsv_camera_feed.py" name="birds_eye_view_node" output="screen">
    </node>
</launch>

